example.py

```diff
@@ -1,7 +1,7 @@
 # PostHog Python library example
 
 # Import the library
-import time
+# import time
```

Comment:
we use this in the example but my editor is chirping me for keeping it in so commenting it out